packages/e2e-tests/jest.config.js

```diff
@@ -9,4 +9,7 @@ module.exports = {
 		'@wordpress/jest-puppeteer-axe',
 		'expect-puppeteer',
 	],
+	testPathIgnorePatterns: [
+		'e2e-tests/specs/performance.test.js',
```

Comment:
Can we put it into its own folder to avoid this ignore rule?

`e2e-tests/benchmark/index.js` - should do the trick